application/routes/vnc.py

```diff
@@ -26,7 +26,8 @@
 from .. import api, app, profiles
 from ..codes import ICtrlStep, ICtrlError, ConnectionType
 from ..utils import int_to_bytes
-
+import logging
+logger = logging.getLogger(__name__)
```

Comment:
Leave one empty line before the first code statement and the import statements. i.e.,
```
...
from ..utils import int_to_bytes

logger = logging.getLogger(__name__)
```